docs/DeveloperGuide.md

```diff
@@ -311,60 +320,68 @@ Step 5. Finally, a `CommandResult` object is created and returned to `LogicManag
     * Pros: Will be easier for the user especially when the list is huge.
     * Cons: There are items with the same name but in different location, will cause confusion.
 
-### Find Item `find KEYWORD [MORE_KEYWORDS]` or `find */KEYWORD [MORE_KEYWORDS]`
+### Find Feature `find`
 
-#### Actual Implementation
+The `find` feature is helps users find and display all items whose names
+contain any of the given keywords, either in full or partial.
 
-The `find KEYWORD [MORE_KEYWORDS]` and `find */KEYWORD [MORE_KEYWORDS]` features find and display all items whose names
-contain any of the given keywords, either in full or partially respectively.
+`find KEYWORD [MORE_KEYWORDS]` displays items whose names contain any of the given keywords in full.
 
-The Sequence Diagram below shows how the components interact with each other for the scenario where the user
-keys in the command `find Chocolate`:
-![Interactions Inside the Logic Component for the `find KEYWORD [MORE_KEYWORDS]` Command](images/FindFullSequenceDiagram.png)
+`find */KEYWORD [MORE_KEYWORDS]` display all items whose names contain any of the given partial keywords .
 
-The Sequence Diagram below shows how the components interact with each other for the scenario where the user
+#### Actual Implementation
+
+The sequence diagram below shows how the components interact with each other for the scenario where the user
 keys in the command `find */cheese egg`:
-![Interactions Inside the Logic Component for the `find */KEYWORD [MORE_KEYWORDS]` Command](images/FindPartialSequenceDiagram.png)
 
-Given below is an example usage scenario and how the find operation behaves at each step.
+<br>
 
-Step 1. The user execute `find Chocolate` to find all the items in the inventory whose names fully match the keyword.
-`StoreMandoParser` takes in the user input and determines the command word (find) and argument ("Chocolate") respectively.
+![Interactions Inside the Logic Component for the `find */KEYWORD [MORE_KEYWORDS]` Command](images/FindPartialSequenceDiagram.png)
```

Comment:
Things inside [] should standardise with the rest

Reply:
Done!